spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,48 @@
+require 'beaker-rspec'
+
+# Various workarounds for Beaker's lack of Puppet 4 support.
+
+def install_puppet_on(host)
+  on host, 'yum -y localinstall http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm'
+  on host, 'yum -y install puppet-agent'
+end
+
+def copy_modules_to(host, opts = {})
+  Dir["#{opts[:source]}/*"].each do |dir|
+    if File.symlink?(dir)
+      scp_to host, dir, opts[:module_dir], {:ignore => 'spec/fixtures/modules'}
+    else
+      scp_to host, dir, opts[:dist_dir]
+    end
+  end
+end
+
+RSpec.configure do |c|
+  # Project root
+  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+
+  # Readable test descriptions
+  c.formatter = :documentation
+
+  # Configure all nodes in nodeset
+  c.before :suite do
+
+    shell('/bin/yum update lvm2 -y')
+
+    system('bundle exec rake spec_prep')
+
+    hosts.each do |host|
+      install_puppet_on(host)
+
+      copy_modules_to(host, {
+        :source     => proj_root + '/spec/fixtures/modules',
```

Comment:
Why copy the fixture version rather than the live version from the project root?